index.js

```diff
@@ -40,9 +40,10 @@ var JSCSFilter = function(inputTree, _options) {
   }
 
   if (this.enabled) {
-    this.rules = config.load(this.configPath || '.jscsrc') || this.config || {};
+    this.rules = config.load(this.configPath || (inputTree + '/.jscsrc')) || this.config || {};
```

Comment:
argh, that's what I was afraid of. your jshint plugin seems to use `inputPaths[0]` instead, but it seems that that isn't available at this point...